README.md

```diff
@@ -64,10 +66,9 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+-Original design idea by Nelson Sakwa on Behance.
+
 
```

Comment:
- [ ] (Optional) It would be nice to include the link to the template design so that people who visit your repository can have easy access to the template design used for this project. 👍

Reply:
- [OPTIONAL ] Great job giving appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). :partying_face::+1: what do you think to add a link?  🤔 so that the user can have the ability to visit the original design.
-  eg: Original design idea by [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) on Behance.